spec.html

```diff
@@ -18504,7 +18508,7 @@ <h1>
           <dd>It determines if its argument is a valid regular expression literal.</dd>
         </dl>
         <emu-alg>
-          1. If FlagText of _literal_ contains any code points other than `d`, `g`, `i`, `m`, `s`, `u`, or `y`, or if it contains the same code point more than once, return *false*.
+          1. If FlagText of _literal_ contains any code points other than `d`, `g`, `i`, `m`, `s`, `u`, or `y`, or _literal_ contains any code point more than once, return *false*.
```

Comment:
I agree, and the proposed change doesn't even appear to be correct right now because the part after "or" references "_literal_" directly rather than "FlagText of _literal_".
```suggestion
          1. Let _flags_ be FlagText of _literal_.
          1. If _flags_ contains any code point other than `d`, `g`, `i`, `m`, `s`, `u`, or `y`, or if _flags_ contains any duplicate code points, return *false*.
```